acme/acme/standalone_test.py

```diff
@@ -88,8 +88,8 @@ def setUp(self):
         }
         from acme.standalone import DVSNIServer
         self.server = DVSNIServer(("", 0), certs=self.certs)
-        # pylint: disable=no-member
-        self.thread = threading.Thread(target=self.server.handle_request)
+        # use serve_forever2() instead of handle_request() to avoid hanging
```

Comment:
how does it help? Moving from one-off (handle_request) to a loop (serve_forever)?